haystack/nodes/label_generator/pseudo_label_generator.py

```diff
@@ -29,12 +29,11 @@ class PseudoLabelGenerator(BaseComponent):
     For example:
 
     ```python
-    |   document_store = DocumentStore(...)
-    |   retriever = Retriever(...)
-    |   qg = QuestionGenerator(model_name_or_path="doc2query/msmarco-t5-base-v1")
-    |   plg = PseudoLabelGenerator(qg, retriever)
-    |   output, output_id = psg.run(documents=document_store.get_all_documents())
-    |
+    document_store = DocumentStore(...)
+    retriever = Retriever(...)
```

Comment:
Let's use existing classes
```suggestion
    document_store = ElasticsearchDocumentStore(...)
    retriever = BM25Retriever(...)
```